meson.build

```diff
@@ -0,0 +1,7 @@
+project('boost-headers', ['cpp'], version: '1.78')
+boost_dep = declare_dependency(include_directories: include_directories('include'))
+# supports meson >= version 0.54
```

Comment:
You can enforce this by setting `meson_version: '>=0.54.0'` as kwargs to the project() function. :)

This both documents the requirement and ensures that anyone running an older version of Meson gets an error message which looks something like:
```
Meson version is 0.49.0 but project requires 0.54.0
```